src/services/config-store.js

```diff
@@ -47,6 +47,18 @@ class ConfigStore {
       throw new Error('Your envSecret seems incorrect (64 characters required). Please check it is correctly set in your .env file.');
     }
 
+    if (!options.connections || !options.connections.constructor.toString().match('Object')) {
+      throw new Error('The connections option seems incorectly set. Please check it is an object of named connections.');
```

Comment:
```suggestion
      throw new Error('The connections option seems incorrectly set. Please check it is an object of named connections.');
```

Reply:
[Question] Do we allow `connections: {}` ? From what I can read in the test we do (And we will not trigger any warning/error), and I'm not sure that's what we want ? WDYT ?